admin/services/config.service.js

```diff
@@ -19,8 +20,12 @@ const configService = {
     let loadingTime = TIME_BETWEEN_QUESTIONS
 
     const timeSettings = await settingDataService.sqlFindOne()
-
-    if (timeSettings) {
+    const group = await groupDataService.sqlFindOneBySchoolId(pupil.school_id)
+    const hasGroupTimeLimits = group && group.loadingTimeLimit && group.questionTimeLimit
```

Comment:
you cant look up the pupil group like this.  There can be many groups in a school, and the pupil may or may not be in one of them.  You should be looking up the group and its settings with a JOIN on the pupilGroup table